python/paddle/incubate/optimizer/lbfgs.py

```diff
@@ -226,7 +232,7 @@ def _directional_evaluate(self, closure, x, alpha, d):
         self._set_param(x)
         return loss, flat_grad
 
-    def step(self, closure):
+    def step(self, closure: Callable) -> Tensor:
```

Comment:
```suggestion
    def step(self, closure: Callable[[], _T]) -> _T:
```

`_T = TypeVar('_T', covariant=True)`